web-app/js/portal/search/GeoSelectionPanel.js

```diff
@@ -106,6 +108,7 @@ Portal.search.GeoSelectionPanel = Ext.extend(Ext.Panel, {
 
     _clearFacetGeometry: function() {
         this.facetMap.clearGeometry();
+        this.clearButton.setDisabled(!this.facetMap.hasCurrentFeature());
```

Comment:
Would it be more clear just to say `this.clearButton.setDisabled(true);` in this instance? or is there a time when it might not evaluate to true?